src/edit/iconsets/KeymapperIconSet.js

```diff
@@ -0,0 +1,7 @@
+L.KeymapperIconSet = L.IconSet.extend({
+
+  _symbols:
+      // eslint-disable-next-line max-len
```

Comment:
Not a big deal but since you'll make another commit, would you mind using two space indents everywhere you can? It helps to slowly make the codebase consistent. CC @7malikk as well, it's not critical but nice to have a standard. Thanks!

Reply:
@jywarren, absolutely I will be happy to stick to this rather nice standard. I believe @7malikk will do same. Many thanks!